Bellerophon/Bellerophon/Sources/BPManager.swift

```diff
@@ -67,60 +89,48 @@ public class BellerophonManager: NSObject {
                 // An error occurred
                 completionHandler(result: .Failed)
             }
-        })
+        }
     }
 
     // MARK: Private Methods
-    override init() {
-        super.init()
-        NSNotificationCenter.defaultCenter().addObserver(self, selector: Selector(stopTimer()), name: UIApplicationDidEnterBackgroundNotification, object: nil)
-    }
-
-    deinit {
-        NSNotificationCenter.defaultCenter().removeObserver(self)
-    }
-
-    func handleAppStatus(status: BellerophonStatusProtocol) {
+    private func handleAppStatus(status: BellerophonStatusProtocol) {
         if status.apiInactive() {
-            self.displayKillSwitch(status.userMessage())
```

Comment:
The model conforming to `BPObservable` needs to provide a user message for your custom view, the reason why we have this message is to force developers providing a user message for their view. The custom view will have to get the data from this model and display it on the screen.

I don't think we should provide a default message because I'm pretty sure 0% of the developers will use it, and it will have to be translated into different languages too which we don't really want to do for now.